### ISSUE TITLE
Change the verb in Prompt limit exceeded message

### ISSUE THREAD

**Issue by VolhaBazhkova** [epam/ai-dial-chat#2482]
### EPAM AI DIAL chat version

0.20.0

### What happened?

Please update the text of the message: to **select** is incorrectly used here. to **insert** is better option
![image](https://github.com/user-attachments/assets/478f0810-73cd-4243-a8b5-92abcdfd1bca)

### Confidential information

- [X] I confirm that do not share any confidential information

**Comment by lattaai1** [comment]
Hello, I've fixed this issue!

This is what I've done:
The solution involves updating the text in the AttachButton component from 'select' to 'insert' for better clarity. This change is made in the `apps/chat/src/components/Files/AttachButton.tsx` file.

I also created pull request: https://github.com/epam/ai-dial-chat/pull/2483

> [!CAUTION]  
> Disclaimer: This fix was created by Latta AI and you should never merge before you check the correctness of generated code!

---
This bug was fixed for free by Latta AI - https://latta.ai/ourmission
